lib/php/webhook-bundle/Resources/config/services.yaml

```diff
@@ -4,7 +4,7 @@ services:
         autoconfigure: true
 
     alchemy.webhook_client:
-        class: GuzzleHttp\Client
+        class: Symfony\Contracts\HttpClient\HttpClientInterface
```

Comment:
You need to configure the Symfony client in the Framework bundle. You won't need this service anymore.
See `Alchemy\ReportBundle\AlchemyReportBundle::prependExtension` to understand how it is done.